packages/backend-socketio/src/schemas/game-data-schema.ts

```diff
@@ -0,0 +1,55 @@
+import * as TripleTriad from '@tripletriad/game';
+import { Board, Card } from '@tripletriad/game/src/lib/common-types';
+import { toZod } from 'tozod';
+import { z } from 'zod';
+
+export const PlayerLabelSchema = z.enum([
+  TripleTriad.PlayerLabel.One,
+  TripleTriad.PlayerLabel.Two,
+]);
```

Comment:
Looks like zod recommends native enums here: https://zod.dev/?id=native-enums

Reply:
We could always move away from enums too, I think they're not a create idea in TS.